test/postman/Auth.postman_collection.json

```diff
@@ -0,0 +1,225 @@
+{
+	"info": {
+		"_postman_id": "ba41f2eb-d7d7-4ca7-bf20-e216e09c9e18",
+		"name": "OSMT-64-Auth",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "3748969"
+	},
+	"item": [
+		{
+			"name": "Okta Session Token",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Receive Okta session token\", function () {",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.sessionToken).exist;",
+							"});",
+							"",
+							"pm.environment.set(\"oktaSessionToken\", pm.response.json().sessionToken);"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disabledSystemHeaders": {
+					"accept": true,
+					"content-type": true
+				}
+			},
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"type": "text"
+					},
+					{
+						"key": "Accept",
+						"value": "application/json",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"username\": \"{{oktaUsername}}\",\n    \"password\": \"{{oktaPassword}}\",\n    \"options\": {\n        \"multiOptionalFactorEnroll\": false,\n        \"warnBeforePasswordExpired\": true\n    }\n}"
+				},
+				"url": {
+					"raw": "https://dev-86780967.okta.com/api/v1/authn",
```

Comment:
Hardcoded values are preventing my local execution. This and "host" (below) should be variables.